packages/lab/src/date-picker/DatePicker.tsx

```diff
@@ -124,6 +125,7 @@ export const DatePicker = forwardRef<HTMLDivElement, DatePickerProps>(
       validationStatus,
       onSelectionChange,
       onChange,
+      visibleMonths,
```

Comment:
I have a hunch that if you don't specify any visibleMonths you get 2...
That's not explicit in code, it's because of 
 `const compact = visibleMonths === 1;`
 
 Also... NIT++, I have been following this PR and I use the commit messages to see what's changed since I last reviewed but that's harder if we don't treat commits message as documentation of not just why but what. It's not much effort to annotate a change and say what's changed.. I can then look through commits without having to click through and diff. I know we have changesets but this PR does not have one (yet) so I rely on the commits